ignite/metrics/js_divergence.py

```diff
@@ -71,14 +72,24 @@ class JSDivergence(KLDivergence):
     """
 
     def _update(self, y_pred: torch.Tensor, y: torch.Tensor) -> None:
-        m_prob = (F.softmax(y_pred, dim=1) + F.softmax(y, dim=1)) / 2
+        y_pred_prob = F.softmax(y_pred, dim=1)
+        y_prob = F.softmax(y, dim=1)
+        m_prob = (y_pred_prob + y_prob) / 2
         m_log = m_prob.log()
-        y_pred = F.log_softmax(y_pred, dim=1)
-        y = F.log_softmax(y, dim=1)
-        self._sum_of_kl += (
-            F.kl_div(m_log, y_pred, log_target=True, reduction="sum")
-            + F.kl_div(m_log, y, log_target=True, reduction="sum")
-        ).to(self._device)
+
+        if Version(torch.__version__) >= Version("1.6.0"):
```

Comment:
Do you think we can gain in perfs if we factorize this check in the global space:
```python

TORCH_VERSION_GT_160 = Version(torch.__version__) >= Version("1.6.0")

class JSDivergence(...):
    ...

    def _update(...):
        ...
        if TORCH_VERSION_GT_160:
             ...
```